pulpito/templates/run.html

```diff
@@ -80,6 +80,8 @@ <h2>
             </div>
             <button type="button" class="btn btn-info {{ '' if run.results.fail or run.results.dead else 'disabled'}}" id="expand-fail-btn"></button>
             <button type="button" class="btn btn-info" id="expand-desc-btn"></button>
+            <button type="button" class="btn btn-info" id="scrape" onclick="window.location.href='http://qa-proxy.ceph.com/teuthology/yuriw-2021-03-23_21:23:36-rados-wip-yuri7-testing-2021-03-23-1015-nautilus-distro-basic-smithi/scrape.log'">Scrape</button>
```

Comment:
@hkaur008 the url I passed was an example and what I was intending to say that we need to url link based on 
the job for pulpito examination please inspect https://pulpito.ceph.com developer tools and see Log file,

job.log_href looks like: http://qa-proxy.ceph.com/teuthology/ideepika-2021-10-14_07:14:29-rbd:iscsi-wip-nizamial09-testing-2021-10-11-1540-distro-default-smithi/6442773/teuthology.log
you can achieve something like that using  scrape_log = "http://qa-proxy.ceph.com/{{ job['name'] }}/scrape.log"

place to look at:  pulpito/templates/run_jobs_table.html

Reply:
soo i just need to add the scrape.log link before the `See detail view` and that link would be like `http://qa-proxy.ceph.com/{{ run.name }}/scrape.log`

Reply:
Okay Got it replaced with a generalised link rather than scrape.log